app/boards/imx8mp_evk_mimx8ml8_adsp.overlay

```diff
@@ -12,6 +12,10 @@
 	};
```

Comment:
@SurajSonawane2415  we should not enable DAI_VIRTUAL and DMA_EMUL by default so please remove them from the config file.